plugins/janus_sip.c

```diff
@@ -6694,6 +6836,7 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
 				SIPTAG_SUPPORTED(NULL),
 				NTATAG_CANCEL_2543(session->account.rfc2543_cancel),
 				TAG_NULL());
+	janus_mutex_unlock(&session->stack->smutex);
```

Comment:
Why these mutexes?

Reply:
Because we access session->stack->s_nua from different thread.